src/status_im/translations/nl.cljs

```diff
@@ -8,11 +8,23 @@
    :chat-name                             "Chatnaam"
    :notifications-title                   "Meldingen en geluiden"
    :offline                               "Offline"
+   :search-for                            "Zoek naar..."
+   :cancel                                "Annuleren"
+   :next                                  "Volgende"
+   :type-a-message                        "Typ een bericht..."
+   :type-a-command                        "Typ een commando..."
+   :error                                 "Error"
+
+   :camera-access-error                   "Om de benodigde cameratoegang te krijgen kan je naar de instellingen gaan en Status > Camera inschakelen."
+   :photos-access-error                   "Om de benodigde galerijtoegang te krijgen kan je naar de instellingen gaan en Status > Foto's inschakelen."
```

Comment:
`Om foto toegang te geven, ga je naar systeem instellingen en zorg je dat Status > Foto's geselecteerd is.`

Would be a more appropriate message since it's an error.